frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -209,224 +209,201 @@ function roleSelected(evt: any) {
+                            />
+                            <label class="mx-2" for="idirSelect">IDIR</label>
+                        </div>
+                        <div class="px-0">
+                            <RadioButton
+                                v-model="formData.domain"
+                                inputId="becidSelect"
+                                name="domainRadioOptions"
+                                :value="domainOptions.BCEID"
+                                @change="userDomainChange()"
+                            />
+                            <label class="mx-2" for="becidSelect">BCeID</label>
                         </div>
                     </div>
 
-                    <div class="form-group">
-                        <div class="row mb-0">
-                            <label for="userIdInput" class="control-label px-0"
-                                >Type user's domain name
-                                <span class="text-danger"> *</span></label
+                    <div class="row">
+                        <label for="userIdInput"
+                            >Type user's id
+                            <span class="text-danger"> *</span></label
+                        >
+                        <div class="mt-0 col-md-3 px-0">
+                            <Field
+                                name="userId"
+                                :validateOnChange="true"
+                                v-model="formData.userId"
+                                v-slot="{ field }"
                             >
-                        </div>
-
-                        <div class="row mt-0">
-                            <div class="col-md-3 px-0">
-                                <Field
+                                <InputText
                                     id="userIdInput"
-                                    class="form-control"
-                                    name="userId"
+                                    :placeholder="
+                                        formData.domain === 'I'
+                                            ? 'Type user\'s IDIR'
+                                            : 'Type user\'s BCeID'
+                                    "
                                     :validateOnChange="true"
-                                    v-model="formData.userId"
-                                    v-slot="{ field }"
-                                >
-                                    <InputText
-                                        :placeholder="
-                                            formData.domain === 'I'
-                                                ? 'Type user\'s IDIR'
-                                                : 'Type user\'s BCeID'
-                                        "
-                                        :validateOnChange="true"
-                                        class="w-100 p-inputtext"
```

Comment:
For all primevue components, by default it already has a classname, we don't need to assign the default classname again. So the "p-inputtext" classname is already there by default, no need to add it again.

Reply:
Agreed!